src/views/virtualmachinesinstance/list/components/VirtualMachineInstanceBreadcrumb/VirtualMachineInstanceBreadcrumb.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { VirtualMachineInstanceModelRef } from '@kubevirt-ui/kubevirt-api/console';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { Breadcrumb, BreadcrumbItem, Button } from '@patternfly/react-core';
+
+const VirtualMachineInstanceBreadcrumb: React.FC<{ namespace: string }> = React.memo(
```

Comment:
Another way to solve this issue can be to have a:
`DetailsPageBreadcrumb` component that you can use from any details page header.
than you will only need one method, and call it with more props, e.g.
```
<DetailsPageBreadcrumb namespace="default" kind="VirtualMachineInstance" modelref={VirtualMachineInstanceModelRef} />
```